### ISSUE TITLE
Need sample project to showcase comparison between cucumber-jvm-parallel-plugin vs this plugin

### ISSUE THREAD

**Issue by vikramvi** [trivago/cucable-plugin#5]
Hi Benjamin,

It will be useful if you can please share very simple project which showcase comparison between cucumber-jvm-parallel-plugin vs this plugin

The sample project can have couple of feature files which just opens browser and goto google/facebook/yahoo.com

I had posted query http://tech.trivago.com/2017/06/30/cucable-maven-plugin-for-parallel-execution-of-cucumber-scenarios/#comment-3473516679

Thanks in advance.

**Comment by laxersaz** [comment]
I don't think a sample project is really necessary because both projects have a pretty clear documentation in my opinion.

The main difference is that the cucumber-jvm-parallel-plugin generates runners for each feature file and Cucable generates runners for each individual scenario within a feature file.
So as an example, if you have 2 feature files with 5 scenarios each, cucumber-jvm-parallel-plugin would generate 2 runners whereas Cucable would generate 10. This is much better in terms of parallel distribution.

Basically it is the same mechanism that the cucumber slices plugin uses (https://github.com/DisneyStudios/cucumber-slices-maven-plugin) - Cucable just uses another method for generating the files (using the official Gherkin parser of Cucumber).

**Comment by laxersaz** [comment]
@vikramvi What do you think? If you need a sample project I can of course do it.

**Comment by vikramvi** [comment]
@laxersaz  It will be nice to have sample project IMHO as first time user may get confused.

Thanks in advance.

**Comment by vikramvi** [comment]
@laxersaz gentle reminder

**Comment by laxersaz** [comment]
Seriously? Sending me a friendly reminder for something that you requested 3 days ago and that is a free open source project that I maintain in my free time is not appropriate. I will get to that eventually when the more urgent feature tickets are closed.

**Comment by laxersaz** [comment]
@vikramvi why did you close this?

**Comment by vikramvi** [comment]
@laxersaz because of your last comment https://github.com/trivago/cucable-plugin/issues/5#issuecomment-326283185
